test/image/mocks/funnel_and_bar.json

```diff
@@ -0,0 +1,55 @@
+{
+    "data": [
```

Comment:
Looks like there's already one pretty complete `funnel_axis_with_other_traces` mock. Can we remove `funnel_and_bar`, `funnel_and_histogram` and `funnel_line`? Or do these mocks lock down something in particular?

Reply:
Yes we can remove funnel and `bar`, `waterfall` and `line` but not the `histogram` for the moment.
I don't know why the histogram didn't properly show up on `funnel_axis_with_other_traces` setup.

Reply:
[Codepen](https://codepen.io/MojtabaSamimi/pen/eoojYo?editors=0010) please notice how the orange `histogram` has bars of equal size!

Reply:
> I don't know why the histogram didn't properly show up on

_not_ probably as in the histogram bars have gaps in-between them?

Reply:
Oh no it's worse, the histogram bar height are wrong. It must be a conflict between the `bar` and `bar-like` categories.

Reply:
> Looks like there's already one pretty complete `funnel_axis_with_other_traces` mock. Can we remove `funnel_and_bar`, `funnel_and_histogram` and `funnel_line`? Or do these mocks lock down something in particular?

Right. Reduced mocks in https://github.com/plotly/plotly.js/pull/3817/commits/1434148694d331480025d49501557a167a9be76c.

Reply:
Ok, thanks. I'll keep this the thread unresolved as the `histogram` + `funnel` problem appears to still be a thing.

Reply:
Histogram case is added in https://github.com/plotly/plotly.js/pull/3817/commits/95076408fb63493313b8c3757ebe2966c521ea8a.

Reply:
@archmoj in https://github.com/plotly/plotly.js/commit/95076408fb63493313b8c3757ebe2966c521ea8a, it seems that the histogram trace coexist properly with funnel traces. Is that correct? In other words, are the concerns from

>  It must be a conflict between the bar and bar-like categories.

not founded?

Reply:
That is correct. & I think so.

Reply:
Ok, great. Resolved.